### ISSUE TITLE
[SUGGESTION] Balance the Command !human

### ISSUE THREAD

**Issue by HollowZaraki** [Azurblau/Zombies.Zone#180]
I think everyone can agree that !human is overused, so i would suggest to balance it a bit.
You would have a certain amounts of !human command so you can redeem this certain times.
I would suggest that we would grand players below lvl 15 unlimited amount of !human commands, but as soon they reach lvl 14 they should be limited, like you get 5 free redeems after reaching lvl 15 and then you redeem 1 point gets taken, so if you have 5 redeems left, you redeem oonce and there would be 4 left. That should balance !human pretty well, so there would be again a need to stay human so you dont waste all your redeems. 
And ofc you could get redeems back by winning maps, i would say the more a map is one the less Redeemer Points you get, like winning a hard map gives you 2-5 redeems but when its an easy map you only get 1 redeem.
Also you maybe should get Redeemer Points by getting zombies top scores, like eating most brains and dealing most damage to barricades. So you wont stuck as a zombie for ever because you never win a map.
Also you could add events which e.g. increase the amount of gained Redeemer Points, like when you win a map while this event is going you would get the double amount.

**Comment by bob-b-b** [comment]
I agree with everything except that humans get more redeems if they win a harder map. People could just abuse this by beating semi hard/hard maps by getting the right group of people or using an op strategy and stack up redeems. It should always be only 1 redeem per win.

**Comment by HollowZaraki** [comment]
ofc it wouldnt be that big differenece if we would change to 1 per map, but i think its important to also gain redeems by doing smth as a zombie

**Comment by bob-b-b** [comment]
If you don't use your 3 kill redeem it could save to be used later but then again it's relatively easy to get kills as a zombie might be too easy to get a bunch of redeems to use for later so it would probably have to be limited to only 1 per game as a zombie. But being able to earn them as a zombie already feels a bit dumb as you have already fucked up and died as a human so I don't think you should be able to earn redeems at all. Maybe as a way to reward zombie mains by doing well, they could get a substantial amount of XP once they get like 10 kills as zombie.

**Comment by HollowZaraki** [comment]
but thats exacly the point, i dont want to support z mainning, ofc its fine but i dont want to give them more reason to go z main. So giving extra exp would be a reward for going z main. But getting redeems just gives you the chance to improve as human the next time
